packages/web/src/components/background-animations/MusicConfetti.tsx

```diff
@@ -157,46 +86,20 @@ const UnconditionalMusicConfetti = (props: Required<MusicConfettiProps>) => {
     ]
   )
 
-  // When we unmount, stop the canvas animation
-  useEffect(() => {
-    return () => {
-      if (confettiPromiseRef.current) {
-        confettiPromiseRef.current.then((animation) =>
-          animation ? animation.stop() : null
-        )
-      }
-    }
-  }, [])
-
-  const [sizing, setSize] = useState({})
-
-  useEffect(() => {
-    setSize({
-      width: window.innerWidth,
-      height: window.innerHeight
-    })
-  }, [])
-
-  useOnResizeEffect(() => {
-    setSize({
-      width: window.innerWidth,
-      height: window.innerHeight
-    })
-  })
-
   return (
     <canvas
       ref={setCanvasRef}
-      {...sizing}
+      width={window.innerWidth}
+      height={window.innerHeight}
       style={{
         pointerEvents: 'none',
         position: 'absolute',
         top: 0,
         left: 0,
-        zIndex: props.zIndex,
+        zIndex: zIndex || 16,
```

Comment:
Is 16 arbitrary or is it meant to be some other item's zIndex +1? We have a[ zIndex file](https://github.com/AudiusProject/audius-client/blob/f94ebec5d626d8fcd4e376345b32dd690b710e22/packages/web/src/utils/zIndex.ts#L5-L6) that you could pull a constant from and reference here if we wants this to always live above certain elements

Reply:
👍